### PR TITLE
Update installer dependencies

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: debian/stretch/foreman-installer/control [hunk 1]
```diff
@@ -13,7 +14,7 @@ Package: foreman-installer
 Architecture: all
 XB-Ruby-Versions: ${ruby:Versions}
 Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter,
-         ruby-kafo (>= 3.0.0),
+         ruby-kafo (>= 4.0.0), ruby-kafo (<< 5.0.0)
```
Comment (mmoll) [inline debian/stretch/foreman-installer/control]: missing comma

File: debian/xenial/foreman-installer/control [hunk 2]
```diff
@@ -13,7 +14,7 @@ Package: foreman-installer
 Architecture: all
 XB-Ruby-Versions: ${ruby:Versions}
 Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter,
-         ruby-kafo (>= 3.0.0),
+         ruby-kafo (>= 4.0.0), ruby-kafo (<< 5.0.0)
```
Comment (mmoll) [inline debian/xenial/foreman-installer/control]: missing comma

**Comment by ekohl** [comment]
Updated

**Comment by mmoll** [comment]
merged, bedankt @ekohl!

**Pull request merged**